be/src/service/backend_service.cpp

```diff
@@ -312,4 +314,25 @@ void BackendService::close_scanner(TScanCloseResult& result_, const TScanClosePa
     result_.status = t_status;
 }
 
+void BackendService::get_stream_load_record(TStreamLoadRecordResult& result, const int64_t last_stream_record_time) {
```

Comment:
Use `TStreamLoadRecordResult* result` as pass out parameter

Reply:
> Use `TStreamLoadRecordResult* result` as pass out parameter

The function interface should be consistent with the thrift generated code.